basket/base/templates/admin/dsar-info.html

```diff
@@ -0,0 +1,183 @@
+      {% if dsar_form.non_field_errors %}
+        {% for error in dsar_form.non_field_errors %}
+          <p class="errornote">
+            {{ error }}
+          </p>
+        {% endfor %}
+      {% endif %}
+
+      <form method="post">
+        {% csrf_token %}
+        <h2>{{ dsar_form.email.label_tag }}</h2>
+        {% if dsar_form.email.errors %}
+          <div class="errornote">{{ dsar_form.email.errors }}</div>
+        {% endif %}
+        <div>{{ dsar_form.email }}</div>
+        <div class="submit-row">
+          <input type="submit" value="Fetch">
+        </div>
+      </form>
+    </div>
+
+    <div class="output-container">
+      {% if dsar_contact %}
+        <h2>User Info:</h2>
+        <table>
+            <tr>
+                <th>Primary Email</th>
+                <td>{{ dsar_contact.email.primary_email }}</td>
+            </tr>
+            <tr>
+                <th>Basket Token</th>
+                <td><a href="https://www.mozilla.org/newsletter/existing/{{ dsar_contact.email.basket_token }}/" target="_blank" title="Open the user's Email Preferences">{{ dsar_contact.email.basket_token }}</a></td>
+            </tr>
+            {% if dsar_contact.email.first_name %}
+            <tr>
+                <th>Name</th>
+                <td>{{ dsar_contact.email.first_name }} {{ dsar_contact.email.last_name }}</td>
+            </tr>
+            {% endif %}
+            <tr>
+                <th>Language</th>
+                <td>{{ dsar_contact.email.email_lang }}</td>
+            </tr>
+            {% if dsar_contact.email.mailing_country %}
+            <tr>
+                <th>Country</th>
+                <td>{{ dsar_contact.email.mailing_country }}</td>
+            </tr>
+            {% endif %}
+            <tr>
+                <th>FxA ID</th>
+                <td>{{ dsar_contact.fxa.fxa_id }}</td>
+            </tr>
+            {% if dsar_contact.fxa.primary_email %}
+            <tr>
+                <th>FxA Primary Email</th>
+                <td>{{ dsar_contact.fxa.primary_email }}</td>
+            </tr>
+            {% endif %}
+            <tr>
+                <th>MoFo Relevant?</th>
+                <td>{{ dsar_contact.mofo.mofo_relevant|yesno }}</td>
+            </tr>
+            <tr>
+                <th>Double Opt In?</th>
+                <td>{{ dsar_contact.email.double_opt_in|yesno }}</td>
+            </tr>
+            <tr>
+                <th>Opt Out of All Email?</th>
+                <td>{{ dsar_contact.email.has_opted_out_of_email|yesno }}</td>
+            </tr>
+            <tr>
+                <th>Subscriptions</th>
+                <td>
+                    <ul>
+                    {% for nl in newsletter_names %}
+                        <li>{{ nl }}</li>
+                    {% empty %}
+                        <li>No subscriptions</li>
+                    {% endfor %}
+                    </ul>
+                </td>
+            </tr>
+        </table>
+        <br>
```

Comment:
All of the below content looks to be duplicated from the dsar.html template. Perhaps we could pull it out into a template include and then pull it into each template to reduce duplication.